bitnami/dataplatform-bp1/values.yaml

```
@@ -1002,61 +1106,60 @@ wavefront:
       enableRuntimeConfigs: true
       ## @param wavefront.collector.discovery.config [array] Configuration for rules based auto-discovery
       ##
-      config:
+      #config:
```

The linter complains about a missing space here:
```suggestion
      # config:
```